package_manifest.yaml

```diff
@@ -499,6 +501,7 @@ katello_packages:
       build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args }}"
       nightly_package_tito_releaser_args:
         - "--arg jenkins_job=katello-master-release"
+        - "--arg jenkins_job_id=lastBuild"
```

Comment:
Why lastBuild instead of the last successful build (which we default to)?

Reply:
Chicken and egg problem the way those jobs work right now. They both build source and the RPM. However, if we are in this current state, the lastSuccessfulBuild will never point to a usable source. Further, we aren't validating the source, so we just need to most recently generated source.

Reply:
But `katello-installer-nightly-release` triggers a build with an explicit job ID so it'd be a matter of restarting that job. I don't think there's a need to pollute our history with this change and then changing it back. Other reasons for failure might be intermittent network failures. We also had jobs that timed out after 2 hours so had no artifact (more relevant on the installer, but still).